### PR TITLE
feature: Form Field Manager inputs are styled in Sequoia form template

### LINKED ISSUE #4719: feature: form field manager inputs should be styled to be consistent with the Sequoia template
# Feature Request

## User Story
As a user, I want inputs added by form field manager to be consistent with Sequoia template styles so that I have a cohesive donation experience.

## Details
The Form Field Manager addon introduces custom fields of a handful of types to the form. These fields should be styled consistently with the standard fields in the Sequoia template, regardless of where in the form they are rendered.

## Visuals
https://www.figma.com/file/jaVXDA4wjQD9OqTi0l9KQe/Form-Themes?node-id=1882%3A44&viewport=-259%2C1586%2C0.5

## Possible Solution
Review implementation of existing fields. Some fields style rules could be made more generic, to also cover form field manager fields.

## Acceptance Criteria
- [ ] All fields supported by Form Field Manager are styled consistently
- [ ] All field options are respected (width especially)
- [ ] Fields introduced by FFM appear correctly, regardless of where in the form they are rendered (after donation levels, separate page, etc)

### PR DESCRIPTION
**Pull request by henryholtgeerts**
## Description
Resolves #4719 
This PR introduces styling and frontend functionality for inputs rendered by the form fields manager addon. Previously, these inputs were not styled and used default browser styles. Now, through a combination of css, pseudo elements and javascript, the inputs are consistent with other inputs native to the Sequoia form template. 

Because Form Field Manager outputs fields in a non-standard way (nesting inputs within labels) some frontend logic was introduced to listen for changes to these inputs, and add "selected" or "checked" classes to the corresponding label. Ultimately, modifying the means by which FFM renders inputs was beyond the scope of this PR, so a javascript solution was utilized. 

Early on it was suggested that Sequoia should introduce a new "separate step" location for Form Fields. After discussion with @ravinderk, it was determined that this kind of modification to FFM is not currently supported, and wouldn't be feasible to implement on a per template basis.

## Affects
This PR affects Sequoia form template assets (css and js).

## What to test
With form fields manager enabled, test out different field types. Do they behave as expected? Test out different locations for the fields, do they work as expected?

## Screenshots:
PRDemo2

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by DevinWalker** [comment]
@henryholtgeerts have you tested this to ensure that modifying the width of elements works as expected?

File: src/Views/Form/Templates/Sequoia/Sequoia.php [hunk 1]
```diff
@@ -68,8 +68,8 @@ public function loadScripts() {
...
 		$templateOptions['payment_information']['checkout_label'] = ! empty( $templateOptions['payment_information']['checkout_label'] ) ? $templateOptions['payment_information']['checkout_label'] : __( 'Process Donation', 'give' );
 
-		wp_enqueue_style( 'give-google-font-montserrat', 'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap', [], GIVE_VERSION );
-		wp_enqueue_style( 'give-sequoia-template-css', GIVE_PLUGIN_URL . 'assets/dist/css/give-sequoia-template.css', [ 'give-styles' ], GIVE_VERSION );
+		wp_enqueue_style( 'give-google-font-montserrat', 'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap', array(), GIVE_VERSION );
+		wp_enqueue_style( 'give-sequoia-template-css', GIVE_PLUGIN_URL . 'assets/dist/css/give-sequoia-template.css', array( 'give-styles' ), GIVE_VERSION );
```
Comment (ravinderk) [inline src/Views/Form/Templates/Sequoia/Sequoia.php:72]: @henryholtgeerts Can you confirm that these changes made by linters or not?

Comment (ravinderk) [reply src/Views/Form/Templates/Sequoia/Sequoia.php:72]: I notices that `[]` converted to `array()`

**Comment by henryholtgeerts** [comment]
> @henryholtgeerts have you tested this to ensure that modifying the width of elements works as expected?

Tested with all the different width settings, and checked for responsive behavior. Everything works as expected.

Desktop: 
Screen Shot 2020-05-12 at 12 30 54 PM

Mobile: 
Screen Shot 2020-05-12 at 12 31 06 PM

File: src/Views/Form/Templates/Sequoia/Sequoia.php (hunk 1 above)
Comment (henryholtgeerts) [reply src/Views/Form/Templates/Sequoia/Sequoia.php:72]: Can confirm these were added by linter, I've been using short array syntax in my work.

**Comment by DevinWalker** [comment]
@henryholtgeerts this looks good on my end. Merging!

**Pull request merged**